Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper/ImageCropper.cs

```diff
@@ -0,0 +1,385 @@
+                _rightThumb.ManipulationCompleted -= ImageCropperThumb_ManipulationCompleted;
+                _rightThumb.KeyDown -= ImageCropperThumb_KeyDown;
+                _rightThumb.KeyUp -= ImageCropperThumb_KeyUp;
+            }
+
+            if (_upperLeftThumb != null)
+            {
+                _upperLeftThumb.ManipulationDelta -= ImageCropperThumb_ManipulationDelta;
+                _upperLeftThumb.ManipulationCompleted -= ImageCropperThumb_ManipulationCompleted;
+                _upperLeftThumb.KeyDown -= ImageCropperThumb_KeyDown;
+                _upperLeftThumb.KeyUp -= ImageCropperThumb_KeyUp;
+            }
+
+            if (_upperRightThumb != null)
+            {
+                _upperRightThumb.ManipulationDelta -= ImageCropperThumb_ManipulationDelta;
+                _upperRightThumb.ManipulationCompleted -= ImageCropperThumb_ManipulationCompleted;
+                _upperRightThumb.KeyDown -= ImageCropperThumb_KeyDown;
+                _upperRightThumb.KeyUp -= ImageCropperThumb_KeyUp;
+            }
+
+            if (_lowerLeftThumb != null)
+            {
+                _lowerLeftThumb.ManipulationDelta -= ImageCropperThumb_ManipulationDelta;
+                _lowerLeftThumb.ManipulationCompleted -= ImageCropperThumb_ManipulationCompleted;
+                _lowerLeftThumb.KeyDown -= ImageCropperThumb_KeyDown;
+                _lowerLeftThumb.KeyUp -= ImageCropperThumb_KeyUp;
+            }
+
+            if (_lowerRigthThumb != null)
+            {
+                _lowerRigthThumb.ManipulationDelta -= ImageCropperThumb_ManipulationDelta;
+                _lowerRigthThumb.ManipulationCompleted -= ImageCropperThumb_ManipulationCompleted;
+                _lowerRigthThumb.KeyDown -= ImageCropperThumb_KeyDown;
+                _lowerRigthThumb.KeyUp -= ImageCropperThumb_KeyUp;
+            }
+        }
+
+        /// <summary>
+        /// Load an image from a file.
+        /// </summary>
+        /// <param name="imageFile">The image file.</param>
+        /// <returns>Task</returns>
+        public async Task LoadImageFromFile(StorageFile imageFile)
+        {
+            var writeableBitmap = new WriteableBitmap(1, 1);
+            using (var stream = await imageFile.OpenReadAsync())
+            {
+                await writeableBitmap.SetSourceAsync(stream);
+            }
+
+            Source = writeableBitmap;
+        }
+
+        /// <summary>
+        /// Saves the cropped image to a stream with the specified format.
+        /// </summary>
+        /// <param name="stream">The target stream.</param>
+        /// <param name="bitmapFileFormat">the specified format.</param>
+        /// <param name="keepRectangularOutput">Whether to always keep rectangular output.</param>
+        /// <returns>Task</returns>
+        public async Task SaveAsync(IRandomAccessStream stream, BitmapFileFormat bitmapFileFormat, bool keepRectangularOutput)
```

Comment:
```suggestion
        public async Task SaveAsync(IRandomAccessStream stream, BitmapFileFormat bitmapFileFormat, bool keepRectangularOutput = false)
```

Just want to add a default value here and I think this will be great!